modules/ROOT/pages/index.adoc

```diff
@@ -1,88 +1,140 @@
-* xref:mule-runtime::runtime-installation-task.adoc[Installing Standalone Mule 4]
+image::monitor.png[Monitor]
 
-== Managing Integrations
+--
+[discrete]
+=== Monitor
 
-After your integration is running in production, monitor its performance:
+Monitor your APIs and integrations using dashboards, metrics, and visualization.
 
-* xref:monitoring::index.adoc[Anypoint Monitoring]
-* xref:api-functional-monitoring::index.adoc[API Functional Monitoring]
-* xref:runtime-manager::monitoring.adoc[Runtime Monitoring]
+* xref:monitoring::index.adoc[View metrics for integration apps and APIs].
+* xref:api-functional-monitoring::index.adoc[Test the functional behavior and performance of your API].
+* xref:runtime-manager::monitoring.adoc[Monitor your deployed apps].
+--
 
-== Managing Anypoint Platform Features
+== Quick Links
 
-Either during the development cycle or after deployment, you can make changes to your integration:
+[#popular-topics]
+=== Popular Topics
 
-* xref:design-center::index.adoc[Design Center]
-* xref:2.x@api-manager::latest-overview-concept.adoc[API Manager]
-* xref:access-management::index.adoc[Access Management]
+//Date Range 5/01/2022 - 6/01/2022 (omits #1 ranking link to landing page, of course)
+* xref:dataweave::dataweave-cookbook-format-dates.adoc[Format Dates and Times]
+* xref:dataweave::dw-operators.adoc[DataWeave Operators]
+* xref:dataweave::dw-core-functions-filter.adoc[DataWeave Function: filter]
+* xref:dataweave::dw-core-functions-map.adoc[DataWeave Function: map]
+* xref:mule-runtime::batch-processing-concept.adoc[Batch Processing]
+* xref:dataweave::dataweave-functions.adoc[DataWeave Reference]
+* xref:connectors::index.adoc[Connectors]
+// rank #8-10:
+// * xref:mule-runtime::mule-error-concept.adoc[Mule Errors]
+// * xref:mule-runtime::cloudhub-architecture.adoc[CloudHub Architecture]
+// * xref:dataweave::dw-core-functions-contains.adoc[DataWeave Function: contains]
 
-== More Resources
+[#latest-releases]
+=== Latest Releases
+
+////
+removing table until we can automate, keeping hidden here for future reference
+[cols=10;90]
+|===
+|April 11
+|xref:api-functional-monitoring/api-functional-monitoring-release-notes.adoc[API Functional Monitoring 2.1]
+
+|April 11
+|xref:connector/connector-db.adoc#1-13-0[Database Connector 1.13]
+
+|April 2
+|xref:design-center/design-center-release-notes-api_specs.adoc#2-41-0[API Designer (Crowd) Release Notes 2.41]
+
+|April 2
+|xref:api-mocking-service/api-mocking-service-release-notes.adoc#2-38-0[API Mocking Service Release Notes 2.38]
+
+|March 31
+|xref:studio/anypoint-studio-7.12-with-4.4-runtime-release-notes.adoc[Anypoint Studio 7.12]
+|===
+////
 
-* xref:release-notes::index.adoc[Release Notes]
-* https://developer.mulesoft.com/tutorials-and-howtos/getting-started/hello-mule[MuleSoft Developer Tutorial "Getting Started with MuleSoft"^]
-* https://training.mulesoft.com/[Training^]
+xref:#[View all release notes,role=view-all]
 
```

Comment:
[#archived-docs]
=== Archived Documentation

* https://archive.docs.mulesoft.com/[Archived MuleSoft Documentation^]

Reply:
I added the archive link at the bottom (and the left nav):

<img width="1231" alt="Screen Shot 2022-10-12 at 12 45 24 PM" src="https://user-images.githubusercontent.com/10934908/195434359-546c043c-bb21-4251-8bad-a7266adf059e.png">